File: ZeroQ/Model/Group.swift
//
//  Group.swift
//  ZeroQ
//
//  Created by joão victor on 09/02/19.
//  Copyright © 2019 hackthon movile. All rights reserved.
//

import Foundation

struct Group: Codable {
    let name: String
    let event: Event
    let wallets: [Wallet]
}

File: src/main/java/com/movile/zeroQ/event/domain/Group.java
package com.movile.zeroQ.event.domain;

import java.util.List;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.OneToMany;

import com.movile.zeroQ.financial.domain.Wallet;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter @Setter
@AllArgsConstructor
@NoArgsConstructor
public class Group {

	@Id
	@GeneratedValue(strategy=GenerationType.AUTO)
	@Column(name="id_group")
	private Integer id;
	@OneToMany(fetch = FetchType.LAZY)
	private List<Wallet> wallets;
	
}
